test/api/generateAddress/index.ts

```diff
@@ -1,30 +1,209 @@
 import assert from 'assert-diff'
 import responses from '../../fixtures/responses'
 import {TestSuite} from '../../utils'
+import { GenerateAddressOptions } from '../../../src/offline/generate-address'
 const {generateAddress: RESPONSE_FIXTURES} = responses
 
+export const config = {
+  // These tests do not use `address` so without skipping the address iteration, we would just
+  // redundantly run the same test twice, with no changes.
+  // Known issue: The test runner will say that these tests are being run under `[Original Address]`.
```

Comment:
What causes this issue?

Reply:
https://github.com/ripple/ripple-lib/blob/develop/test/ripple-api-test.ts#L36

We currently assume that every method takes an address. I have an idea for fixing this; I might put it in a separate PR.

Reply:
Nevermind - I fixed it here 🎉 

It turns out there were a bunch of other tests that also were being run repeatedly, unnecessarily. Now tests are run once, unless they take an address, in which case they are run first with a classic address and second with an X-address, as intended.